test/jest/victory-errorbars/error-bar.test.js

```diff
@@ -0,0 +1,108 @@
+import { render } from "@testing-library/react";
+import { forEach, omit } from "lodash";
+import React from "react";
+import { ErrorBar } from "victory-errorbar";
+import * as d3Scale from "victory-vendor/d3-scale";
+
+describe("victory-primitives/error-bar", () => {
+  const baseProps = {
+    x: 4,
+    y: 5,
+    errorX: [1, 3],
+    errorY: [0.2, 2],
+    scale: {
+      x: d3Scale.scaleLinear(),
+      y: d3Scale.scaleLinear()
+    },
+    borderWidth: 20
+  };
+
+  const compareLineCoordinates = (line, coordinates) => {
+    forEach(coordinates, (coordinateValue, coordinateName) => {
+      expect(parseFloat(line.getAttribute(coordinateName), 10)).toEqual(
+        parseFloat(coordinateValue, 10)
```

Comment:
`parseFloat` does not accept a radix arg, so it's not necessary here.
```suggestion
      expect(parseFloat(line.getAttribute(coordinateName))).toEqual(
        parseFloat(coordinateValue)
```